regression/pages/whitelabel/const.py

```diff
@@ -8,15 +8,16 @@
 AUTH_USERNAME = os.environ['BASIC_AUTH_USER']
 AUTH_PASSWORD = os.environ['BASIC_AUTH_PASSWORD']
 
-# Get API Access token
+# Global password
+PASSWORD = os.environ.get('GLOBAL_PASSWORD')
```

Comment:
What is this password `GLOBAL_PASSWORD` for? If its basic auth then please rename the variable name.

Reply:
it is user password for all new and old users, we are using one default value for all passwords

Reply:
then maybe we can do this:
```python
DEFAULT_USER_PASSWORD = os.environ.get('DEFAULT_USER_PASSWORD')
```

Reply:
much better name, will do it

Reply:
on second thought, I will let it be for now, changing the environment variables in different environments could be a tough, some other time may be